docs/docs/ingest/configuration-reference.md

```diff
@@ -93,6 +93,42 @@ documentation.
 
 :::
 
+## `VerifiedAnswerStore`
```

Comment:
why is this in the ingest documentation? 

think would make more sense to have a dedicated "Verified Answers" page in the server docs.

in that page, you could also include information on adding the verified answer query to the server